script/s3.js

```diff
@@ -71,6 +75,15 @@ const emptyBucket = (bucketName, callback) => {
 
 const isMainFile = file => file === "index.html";
 
+const isGzip = metaData => {
```

Comment:
I'd rename this to `getMetadataObject`. If named `isGzip` it appears to be a function returning a boolean